react/__mocks__/vtex.twitter-meta-tags.js

```diff
@@ -0,0 +1,3 @@
+import React from 'react'
+
+export const TwitterMetaTags = () => <div>TwitterMetaTags</div>
```

Comment:
Why do you need to do this? I think you can make the twittermetatags component work properly and reuse it. @icazevedo @victorhmp what do you think about it.

Reply:
Since this component only adds the meta tags for a Twitter Card, I believe is only being tested for being added correctly and that's the reason we are using the `manual mock`.
This was based on the `mock definition` for the [ProductOpenGraph](https://github.com/vtex-apps/store/blob/master/react/__mocks__/vtex.open-graph.js)  component that already exists in the code base.

Reply:
You're right, np.